### DISCUSSION TITLE
Not able to start cardanosolutions/cardano-node-ogmios

### DISCUSSION THREAD

**Discussion by santosh** [CardanoSolutions/ogmios#355] [Q & A] [answered]
Following this documentation to start cardano node + ogmios: https://ogmios.dev/getting-started/docker/

```
docker run -it \
  --name cardano-node-ogmios \
  -p 1337:1337 \
  -v cardano-node-ogmios-mainnet-db:/db \
  cardanosolutions/cardano-node-ogmios:latest
```

Container is exiting with this last logs:

```
Conway related : There was an error parsing the genesis file: /config/cardano-node/../genesis/conway.json Error: "Error in $: key \"poolVotingThresholds\" not found"
```

Do you have any idea?

**Comment by KtorZ** [comment, marked as answer]
Yes, the default / official configuration files for the node do not work for versions behind 8.1.2 unless you are connecting to SanchoNet. 

So you either want to: 

- Downgrade the node to 8.1.2
- Use old configuration files from 8.1.2

**Reply by santosh** [reply]
But I'm using docker images to run the container. How can I proceed with your suggestions?

**Reply by KtorZ** [reply]
- download (old) configurations from input-output-hk/cardano-configurations 
- mount the configurations as a volume
- use them within the containers 

I'll try to push patched images in the upcoming with working configurations for all networks as well. I need to patch the official configuration files first though.

**Reply by nstanford5** [reply]
I'm still getting this error -- have the config files been patched? Or should I proceed with the workaround?

**Reply by KtorZ** [reply]
They have been patched, and they broke again with the latest release of the node. At this point, please fill an issue to intersectMBO on the Cardano-node repository. 

It's getting a little bit out of hand to keep track with unparseable configurations provided by the node.

**Reply by nstanford5** [reply]
Do you know the last working version?

**Reply by KtorZ** [reply]
In principle, the confit set that was shipped in the latest release works only for SanchoNet. So configurations from the previous release should work for mainnet and other test networks.
